documentation/Get-PnPWebPermissions.md

```diff
@@ -0,0 +1,76 @@
+---
+Module Name: PnP.PowerShell
+schema: 2.0.0
+applicable: SharePoint Online
+online version: https://pnp.github.io/powershell/cmdlets/Get-PnPWebPermissions.html
+external help file: PnP.PowerShell.dll-Help.xml
+title: Get-PnPWebPermissions
+---
+  
+# Get-PnPWebPermissions
```

Comment:
hi @danielcecil , can you please replace the plural `PnPWebPermissions` to singular `PnPWebPermission` everywhere in this documentation ? 

According to PowerShell convention, we can't have plural in the cmdlet name itself. The `PnPListPermissions` is a mistake as well , I will be making the necessary change there.